AppCenterDistribute/AppCenterDistribute/MSDistributePrivate.h

```diff
@@ -225,32 +230,45 @@ static NSString *const kMSTesterAppUpdateSetupFailedKey = @"MSTesterAppUpdateSet
 - (BOOL)handleUpdate:(MSReleaseDetails *)details;
 
 /**
- * Store details about downloaded release to report it after installation.
+ * Store details about downloaded release.
```

Comment:
Just voicing some thoughts here: Maybe it's just me but my expectation for a comment in a header is that it's a correct sentence with articles, etc.
Do we want to do that in the future?

Reply:
Sorry, my grammar isn't good. Does this look good now?